src/libraries/Directory.Build.targets

```diff
@@ -146,6 +146,13 @@
     <PackageReference Include="Microsoft.DotNet.GenFacades" Version="$(MicrosoftDotNetGenFacadesVersion)" PrivateAssets="all" IsImplicitlyDefined="true" />
   </ItemGroup>
 
+  <Import Sdk="Microsoft.DotNet.PackageValidation" Project="Sdk.targets" Condition="'$(IsSourceProject)' == 'true' and !Exists('$(PkgProjPath)')" />
```

Comment:
adding a pkgproj to just limit it to running for pkgprojs. we can enable it for all the packages without porting them to dotnet pack task. i will enable all the rest of packages in a separate pr